pycaret/time_series/forecasting/oop.py

```diff
@@ -1455,6 +1488,7 @@ def setup(
         self._disable_metrics()
 
         self.logger.info(f"setup() successfully completed in {runtime}s...............")
+        self._mlflow_log_setup()
```

Comment:
You should only log this if the user is logging experiment 

```python
# mlflow logging
if self.logging_param:
  ....
```

Similar to what is done in the OOP file currently

![image](https://user-images.githubusercontent.com/33585645/162339875-ec06382e-8796-4723-8040-427fc27c35a8.png)